specs/test_formats/shuffling/README.md

```diff
@@ -7,26 +7,32 @@ Clients may take different approaches to shuffling, for optimizing,
  and supporting advanced lookup behavior back in older history.
 
 For implementers, possible test runners implementing testing can include:
-1) Just test permute-index, run it for each index `i` in `range(count)`, and check against expected `output[i]` (default spec implementation).
+1) Just test permute-index, run it for each index `i` in `range(count)`, and check against expected `mapping[i]` (default spec implementation).
 2) Test un-permute-index (the reverse lookup; implemented by running the shuffling rounds in reverse, from `round_count-1` to `0`).
 3) Test the optimized complete shuffle, where all indices are shuffled at once; test output in one go.
 4) Test complete shuffle in reverse (reverse rounds, same as #2).
 
 ## Test case format
 
+### `mapping.yaml`
+
 ```yaml
 seed: bytes32
 count: int
-shuffled: List[int]
+mapping: List[int]
 ```
 
-- The `bytes32` is encoded a string, hexadecimal encoding, prefixed with `0x`.
+- The `bytes32` is encoded as a string, hexadecimal encoding, prefixed with `0x`.
```

Comment:
Not having an `.ssz` format here keeps the `0x` parse requirement on prysmatic. Was removing it a goal? is it worth doing an SSZ alternative broken into subfiles

Reply:
It was a goal for full objects. I imagine it's not much of a problem (if at all) for a small non-SSZ metadata object.